ParabolInc/parabol
# chore: Migrate MeetingTemplate table (Phase 2 of 3)

# Description

- [x] Copies the RethinkDB MeetingTemplate table to PG
- [x] Does it all in a single all-or-nothing transaction
- [x] Adds an equality checker to graphql so you can make sure the data in both are.... 

## Demo

https://www.loom.com/share/6b4676756d7445a19a73ac92d6b33045

## Testing scenarios

- see demo
- run migration
- run checkEquality(table: "MeetingTemplate") mutation. see both DBs have the same # of rows
